src/components/Navbar/index.jsx

```diff
@@ -69,34 +72,22 @@ const Navbar = () => {
           ) : (
             <Menu.Item key="8" icon={<UserOutlined />}>
               <NavLink to="/register" activeClassName="active">
-                Register
+                Sign up
               </NavLink>
             </Menu.Item>
           )}
 
           {logStatus ? (
-            <Menu.Item key="9" disabled icon={<SettingOutlined />}>
-              <Button type="button" onClick={disconnection}>
-                Disconnect
-              </Button>
+            <Menu.Item key="9" icon={<LogoutOutlined />}>
+              <NavLink type="button" onClick={disconnection} to="/">
+                Logout
+              </NavLink>
             </Menu.Item>
           ) : (
             <Menu.Item key="1" icon={<UserOutlined />}>
-              <NavLink to="/login" activeClassName="active">
-                Connection
-              </NavLink>
+              <NavLink to="/login">Sign in</NavLink>
```

Comment:
why did you delete the activeClassName attibute ?

Reply:
Readded :)